symengine/derivative.cpp

```diff
@@ -1,19 +1,18 @@
-#include <symengine/basic.h>
-#include <symengine/derivative.h>
```

Comment:
I believe it was redundant because it was alredy #included in some other header files. Also I think I ran into some redefinition issue when compiling.